OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -102,8 +102,9 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 
 			var pos = self.CenterPosition;
 			var mobile = move as Mobile;
-			if (!Target.IsInRange(pos, maxRange) || Target.IsInRange(pos, minRange)
-				|| (mobile != null && !mobile.CanInteractWithGroundLayer(self)))
+			if (!Target.IsInRange(self.CenterPosition, maxRange)
```

Comment:
`self.CenterPosition` is cached as `pos` above, so why do you use `self.CenterPosition` directly?